static/app/components/stream/group.tsx

```diff
@@ -436,16 +436,12 @@ function BaseGroupRow({
 
       {withChart && !displayReprocessingLayout && issueTypeConfig.stats.enabled && (
         <ChartWrapper narrowGroups={narrowGroups}>
-          {!group.filtered?.stats && !group.stats ? (
-            <Placeholder height="24px" />
-          ) : (
-            <GroupChart
-              statsPeriod={statsPeriod!}
-              data={group}
-              showSecondaryPoints={showSecondaryPoints}
-              showMarkLine
-            />
-          )}
+          <GroupChart
```

Comment:
The only visible change to the user is that we remove the chart placeholder and render an empty chart asap. We do this to allow echarts to performs setup asap and not once the content is loaded